app/models/events/comment_announced.rb

```diff
@@ -0,0 +1,29 @@
+class Events::CommentAnnounced < Event
+  include Events::Notify::InApp
+  include Events::Notify::ByEmail
+
+  def self.publish!(model, actor, audiences)
+    super model,
+          user: actor,
+          custom_fields: { audiences: }
```

Comment:
is this valid ruby? Does it mean { audiences: audiences }

Reply:
Yes it is a ruby 3.1 feature - https://github.com/ruby/ruby/commit/c60dbcd1c55cd77a24c41d5e1a9555622be8b2b8